src/main/java/mekanism/common/multipart/PartUniversalCable.java

```diff
@@ -84,15 +84,20 @@ public void update()
 						if(CapabilityUtils.hasCapability(outputter, Capabilities.CABLE_OUTPUTTER_CAPABILITY, side.getOpposite()) && CapabilityUtils.hasCapability(outputter, Capabilities.ENERGY_STORAGE_CAPABILITY, side.getOpposite()))
 						{
 							IStrictEnergyStorage storage = CapabilityUtils.getCapability(outputter, Capabilities.ENERGY_STORAGE_CAPABILITY, side.getOpposite());
-							double received = Math.min(storage.getEnergy(), canDraw);
-							double toDraw = received;
-
-							if(received > 0)
+							
+							if(storage.getEnergy() == null)
```

Comment:
not gonna lie i was just guessing what was wrong because being away from a dev environment sucks